src/web/basics/form-elements/range/range.scss

```diff
@@ -0,0 +1,131 @@
+@import '~basics/fonts/fonts.scss';
+@import '~basics/colors/colors.scss';
+@import '~helpers/css/mixins/base.scss';
+
+.hig__range {
+  position: relative;
+  margin-bottom: 20px;
+  width: 300px;
+
+  font-family: 'ArtifaktElement', sans-serif;
+}
+
+.hig__range__field {
+  -webkit-appearance: none;
+  width: 100%; /* Specific width is required for Firefox. */
+  background-color: transparent; /* Otherwise white in Chrome */
+  box-sizing: content-box;
+  height: 25px;
+  margin: 17px 0 0 0;
+
+  @each $vendor in ('-webkit-slider-thumb', '-moz-range-thumb', '-ms-thumb') {
+    &::#{$vendor} {
+      box-sizing: content-box;
+      -webkit-appearance: none;
+
+      height: 16px;
+      width: 16px;
+      background: color(hig-gray-50);
+      border-radius: 50%;
+      cursor: pointer;
+      position: relative;
+      top: 50%;
+      transform: translateY(-50%);
+      border: 2px solid color(hig-white);
+    }
+
+    &:focus::#{$vendor} {
+      background-color: color(adsk-blue);
+    }
+
+    &:active::#{vendor} {
+      background-color: color(adsk-blue-dark);
+    }
+
+    &[disabled]::#{vendor} {
+      background-color: color(hig-gray-60);
+    }
+
+    &[disabled]::-webkit-slider-thumb{
+      background-color: color(hig-white);
+      border: 2px dotted color(hig-gray-60);
+      outline: 2px solid color(hig-white);
+    }
+  }
```

Comment:
As written the disabled styles aren't applied for Firefox or Microsoft browsers. Let's debug why the interpolated rule for disabled doesn't work. I have some ideas about it & can help.